src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -69,10 +69,6 @@ export interface SavedObjectsCreateOptions extends SavedObjectsBaseOptions {
    * Note: this can only be used for multi-namespace object types.
    */
   initialNamespaces?: string[];
-  /**
-   * workspaces the new created objects belong to
-   */
-  workspaces?: string[];
```

Comment:
what is the intent to remove this field?

Reply:
As there is already a `workspaces` in SavedObjectsBaseOptions and we can delete this duplicate one.